docs/en/getting_started/install.md

```diff
@@ -0,0 +1,153 @@
+# Installation
+
+## System Requirements
+
+ClickHouse can run on any Linux, FreeBSD or Mac OS X with x86\_64, AArch64 or PowerPC64LE CPU architecture.
+
+Though pre-built binaries are typically compiled for x86\_64 and leverage SSE 4.2 instruction set, so unless otherwise stated usage of CPU that supports it becomes an additional system requirement. Here's the command to check if current CPU has support for SSE 4.2:
```

Comment:
Do we distribute binaries that compiled in other way than in x86\_64 SSE 4.2 systems? Can this sentence be rewrited as 
```
If you use ClickHouse official distribution, check that your system has

- x86\_64 architecture.
- Processors that support SSE 4.2 instruction set.

Use the following command to check SSE 4.2 support: